src/main/java/be/seeseemelk/mockbukkit/entity/ArmorStandMock.java

```diff
@@ -38,22 +40,23 @@ public ArmorStandMock(ServerMock server, UUID uuid)
 	}
 
 	@Override
-	public EntityType getType()
+	public @NotNull EntityType getType()
 	{
 		return EntityType.ARMOR_STAND;
 	}
 
 	@Override
+	@NotNull
 	public EntityEquipment getEquipment()
 	{
 		return equipment;
 	}
 
 	@Override
 	@Deprecated
-	public ItemStack getBoots()
+	public @NotNull ItemStack getBoots()
 	{
-		return getEquipment().getBoots();
+		return getEquipment().getItem(EquipmentSlot.FEET);
```

Comment:
Why?

Reply:
getBoots() could returns null variable, getItem(...) not. Another way is to check if getBoots() returns null. If true, return Air